sdk/formrecognizer/Azure.AI.FormRecognizer/README.md

```diff
@@ -542,12 +555,14 @@ for (int i = 0; i < result.Documents.Count; i++)
 }
 ```
 
-You are not limited to invoices! There are a few prebuilt models to choose from, each of which has its own set of supported fields:
+You are not limited to invoices! There are a couple of prebuilt models to choose from, each of which has its own set of supported fields:
 - Analyze business cards using the `prebuilt-businessCard` model. [Supported fields][businessCard_fields].
 - Analyze driver licenses and international passports using the `prebuilt-idDocument` model. [Supported fields][idDocument_fields].
+- Analyze US health insurance cards using the `prebuilt-healthInsuranceCard.us` model.
 - Analyze invoices using the `prebuilt-invoice` model. [Supported fields][invoice_fields].
 - Analyze receipts using the `prebuilt-receipt` model. [Supported fields][receipt_fields].
 - Analyze US W2 tax forms using the `prebuilt-tax.us.w2` model. [Supported fields][w2_fields].
+- Analyze vaccination cards using the `prebuilt-vaccinationCard` model.
```

Comment:
Do we already have convenient aka.ms links to docs for the new prebuilt models? Couldn't find them.